### ISSUE TITLE
Reaction with more than 3 reactants

### ISSUE THREAD

**Issue by oliviaAB** [alanderos91/BioSimulator.jl#32]
Hi,

I'm using Biosimulator for my R package sismonr, to simulate gene expression. My model can involve reactions with more than 3 reactants, e.g.

```
RBS1GCN1reg2GCN1B + RBS1GCN1reg3GCN1B + RBS1GCN1reg5GCN1B --> P2GCN1 + P3GCN1 + P5GCN1
```

In the reaction above, a RNA molecule bound by 3 regulators is degraded, releasing the 3 regulator molecules. The RNA molecule is modelled as the sum of its regulator binding sites, e.g. RBS1GCN1_reg1 + RBS1GCN1_reg2 + RBS1GCN1_reg3. However with the change in BioSimulator v0.9.2, (notably with the function `get_kinetic_law()`), I can't simulate my system. I get the error:

```
ERROR: reaction is not elementary or mass action
```

Is there a way to allow for this type of reaction, as it was possible before?

Thanks!

**Comment by alanderos91** [comment]
Sorry, I should have checked with you first. I forgot you were using higher-order reactions. Will get this working again in a day or two.

Will you need reactions of order > 3 any time in the near future?

**Comment by oliviaAB** [comment]
That's great thank you! Yes my model doesn't have theoretical limits. Do you think that can be a problem? In the meantime I added a line in my code to download v0.5.1 of BioSimulator so there's no rush :)

**Comment by alanderos91** [comment]
This has been addressed in the latest commit ([a2904f2](a2904f296a893930196df66cf4da0d9f98685438)).

I will tag a new release after addressing a couple other issues (e.g. documentation not showing). Feel free to point out any other issues if you happen to test on sismonr.

**Comment by oliviaAB** [comment]
Great thanks, I'll play with it in the next couple of days :) so far so good, the issue about the number of reactants is solved.
